client/src/js/actions/authAction.js

```diff
@@ -0,0 +1,118 @@
+import jwt from 'jsonwebtoken';
+import Api from '../utils/api';
+import {
+  LOGIN_USER_SUCCESS,
+  LOGIN_USER_FAIL,
+  SET_CURRENT_USER,
+  REMOVE_CURRENT_USER,
+  LOGOUT_USER } from '../constants';
+
+
+/**
+ * @param {*} user
+ * @return {*} empty
```

Comment:
Please, add the right data-type of variables in your jsdoc, and also a short description of the variable.